src/Microsoft.VisualStudio.Services.Agent/SecretMasker.cs

```diff
@@ -45,11 +44,12 @@ public string MaskSecrets(string input)
 
             // get indexes and lengths of all substrings that will be replaced
             var positionsToReplace = new List<ReplacementPosition>();
-            foreach (ISecret secretMask in _secretMasks)
+            foreach (ISecret secret in _secrets)
```

Comment:
The "mask" classes were previously renamed to "secret" classes. I fixed these variable names to match.